contrib/mesos/README.md

```diff
@@ -0,0 +1,46 @@
+# Kubernetes-Mesos
+
+Kubernetes-Mesos modifies Kubernetes to act as an [Apache Mesos](http://mesos.apache.org/) framework.
+
+## Features On Mesos
+
+Kubernetes gains the following benefits when installed on Mesos:
+
+- **Auto-Scaling** - Kubernetes minion nodes are created automatically, up to the size of the provisioned Mesos cluster.
+- **Resource Sharing** - Co-location of Kubernetes with other popular next-generation services on the same cluster (e.g. [Hadoop](https://github.com/mesos/hadoop), [Spark](http://spark.apache.org/), and [Chronos](https://mesos.github.io/chronos/), [Cassandra](http://mesosphere.github.io/cassandra-mesos/), etc.). Resources are allocated to the frameworks based on fairness and can be claimed or passed on depending on framework load.
+- **Infrastructure Independence** - Because Mesos is deployable to nearly every IaaS cloud provider, or even your own data center, Kubernetes-Mesos workloads can easily be migrated from one of these environments to another.
+
+## Features On DCOS
+
+Kubernetes can also be installed on [Mesosphere DCOS](https://mesosphere.com/learn/), which runs Mesos as its core. This provides the following *additional* enterprise features:
+
+- **High Availability** - Kubernetes components themselves run within Marathon, which manages restarting/recreating them if they fail.
+- **Easy Installation** - One-step installation via the [DCOS CLI](https://github.com/mesosphere/dcos-cli) or DCOS UI. Both download releases from the [Mesosphere Universe](https://github.com/mesosphere/universe), [Multiverse](https://github.com/mesosphere/multiverse), or private package repositories.
+- **Easy Maintenance** - See what's going on in the cluster with the DCOS UI.
+- **Hybrid Cloud Deployment** - Install across multiple infrastructures and use pod labels and node metadata to customize scheduling.
+
+For more information about how Kubernetes-Mesos is different from Kubernetes, see [Architecture](./docs/architecture.md).
+
+
+## Release Status
+
+Kubernetes-Mesos is alpha quality, still under active development, and not yet recommended for production systems.
+
+For more information about development progress, see the [known issues](./docs/issues.md) or the [kubernetes-mesos repository](https://github.com/mesosphere/kubernetes-mesos) where backlog issues are tracked.
+
+## Usage
+
+This project combines concepts and technologies from two already-complex projects: Mesos and Kubernetes. It may help to familiarize yourself with the basics of each project before reading on:
+
+* [Mesos Documentation](http://mesos.apache.org/documentation/latest)
+* [Kubernetes Documentation](../../README.md) and [Key Concepts](../../DESIGN.md#key-concepts)
```

Comment:
The latest version of the file still lists individual links to the docs on labels, nodes, pods, etc. These docs have been moved into docs/user-guide/ (so your links are no longer up-to-date) but I think you should just not link to them, as I'm sure they'll move around again at some point.